src/main/flight/pid.c

```diff
@@ -515,8 +519,12 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
             // apply filters
             float gyroRateFiltered = dtermNotchFilterApplyFn(dtermFilterNotch[axis], gyroRate);
             gyroRateFiltered = dtermLpfApplyFn(dtermFilterLpf[axis], gyroRateFiltered);
-
-            const float rD = dynCd * MIN(getRcDeflectionAbs(axis) * relaxFactor, 1.0f) * currentPidSetpoint - gyroRateFiltered;    // cr - y
+            float rD;
```

Comment:
Discussed this in slack, just wanted to copy it here so we can discuss alternatives:

``` c
            float scaledRelaxFactor = 1;
            if (relaxFactor > 0.0f) {
                scaledRelaxFactor = MIN(getRcDeflectionAbs(axis) * relaxFactor, 1.0f);
            }
            const float rD = dynCd * scaledRelaxFactor * currentPidSetpoint - gyroRateFiltered;
```